src/shared/seccomp-util.c

```diff
@@ -1032,8 +1033,16 @@ int seccomp_load_syscall_filter_set_raw(uint32_t default_action, Hashmap* set, u
                         int id = PTR_TO_INT(syscall_id) - 1;
                         int error = PTR_TO_INT(val);
 
-                        if (action != SCMP_ACT_ALLOW && error >= 0)
+                        if (error == EXEC_SYSCALL_ERROR_NUMBER_KILL)
+                                a = SCMP_ACT_KILL;
+                        else if (error == EXEC_SYSCALL_ERROR_NUMBER_LOG)
+                                a = SCMP_ACT_LOG;
+                        else if (action != SCMP_ACT_ALLOW && error >= 0)
                                 a = SCMP_ACT_ERRNO(error);
+                        else if (error == EXEC_SYSCALL_ERROR_NUMBER_KILL)
+                                a = SCMP_ACT_KILL;
+                        else if (error == EXEC_SYSCALL_ERROR_NUMBER_LOG)
+                                a = SCMP_ACT_LOG;
```

Comment:
uh? you add each if check twice?

Reply:
whoops, poor squash merging while cross-eyed...